### PR TITLE
[5.7] Simplify split.sh.

### PR DESCRIPTION
**Pull request by lucasmichot**
Simplify `split.sh`.

### CODE REVIEW

File: bin/split.sh [hunk 1]
```diff
@@ -18,60 +18,14 @@ function remote()
...
-remote pipeline git@github.com:illuminate/pipeline.git
-remote queue git@github.com:illuminate/queue.git
-remote redis git@github.com:illuminate/redis.git
-remote routing git@github.com:illuminate/routing.git
-remote session git@github.com:illuminate/session.git
-remote support git@github.com:illuminate/support.git
-remote translation git@github.com:illuminate/translation.git
-remote validation git@github.com:illuminate/validation.git
-remote view git@github.com:illuminate/view.git
+for REMOTE in Broadcasting Bus Cache Config Console Container Contracts Cookie Database Encryption Events Filesystem Hashing Http Log Mail Notifications Pagination Pipeline Queue Redis Routing Session Support Translation Validation View
```
Comment (mfn) [inline bin/split.sh]: One thing about the "old" system I find more useful: you don't have to scroll a list from left to right >100 chars to check something: an alphabetical top-down list is much easier to scan, edit (diffs)

Comment (fitztrev) [reply bin/split.sh]: Another option:

```sh
for REMOTE in \
    Broadcasting \
    Bus \
    Cache \
    Config \
    Console \
    ...
do
    echo $REMOTE
done
```

Comment (lucasmichot) [reply bin/split.sh]: See 4fd8b48555030e46be4171a53619a19731a020bd

**Comment by taylorotwell** [comment]
I'm happy with the scripts as they are.

**Pull request closed without merging**